sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/SessionRecords/AnomalyDetectionConfigurationLiveTests/UpdateDetectionConfigurationWithMinimumSetupAndGetInstance(False).json

```diff
@@ -8,11 +8,8 @@
         "Content-Length": "858",
         "Content-Type": "application/json",
         "Ocp-Apim-Subscription-Key": "Sanitized",
-        "traceparent": "00-627b60a85cf8e240bf2b2731fcbb0dc1-be6fa32a2c746e4f-00",
```

Comment:
Recordings required because the response type changed (before: 204 + empty body; now: 200 + non-empty body).